source/jquery.flot.js

```diff
@@ -2492,10 +2494,33 @@ Licensed under the MIT license.
             }
         }
 
-        // returns the data item the mouse is over/ the cursor is closest to, or null if none is found
+        function findNearbyItems(mouseX, mouseY, seriesFilter, radius, computeDistance) {
+            var items = findItems(mouseX, mouseY, seriesFilter, radius, computeDistance);
+            executeHooks(hooks.findNearbyItems, [mouseX, mouseY, seriesFilter, radius, computeDistance, items]);
+            return items;
+        }
+
         function findNearbyItem(mouseX, mouseY, seriesFilter, radius, computeDistance) {
-            var i, j,
-                item = null,
+            var items = findNearbyItems(mouseX, mouseY, seriesFilter, radius, computeDistance);
+            var closest = items[0];
+ 
+            for (var i = 1; i < items.length; ++i) {
+               if (closest.distance != undefined && items[i].distance < closest.distance) {
+                  closest = items[i];
+               }
+            }
+ 
+            if (closest) {
+                return closest;
+            }
+
+            return null;
+         }
+
+        // returns the data item the mouse is over/ the cursor is closest to, or null if none is found
+        function findItems(mouseX, mouseY, seriesFilter, radius, computeDistance) {
+            var i, j, k, foundItems = [],
+                items = [],
```

Comment:
I know this isn't new code, but can we use a more sane datatype for `items`, like an array of objects?

Reply:
Fixed.